Cargo.toml

```diff
@@ -23,6 +24,8 @@ pretty_assertions = "1.2"
 regex = "1"
 serde = { version = "1.0", features = ["derive"] }
 serde-value = "0.7"
+tokio = { version = "1.20", default-features = false, features = ["macros", "rt"] }
```

Comment:
I know you didn't want to have support for `from_file` functions for the async reader, but I think we should still have at least one test which actually reads from a file, which would require the `fs` feature here. At least in `dev-dependencies` I think it's okay to have more features.

Reply:
Actually, I do not see why we need to test actual reading from a file. I doubt that this test could catch something that is not catched by the test that read from memory. For example, we also does not have tests for sync reader that reads from a file (we have only two examples for that).